app/routes/report/kbMatch/index.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+
+const kbMatchMiddleware = require('../../../middleware/kbMatch');
+
+const kbMatches = require('./kbMatches');
+const kbMatchedStatements = require('./kbMatchedStatements');
+
+const router = express.Router({mergeParams: true});
+
+router.param('kbMatch', kbMatchMiddleware);
+
+router.use('/', kbMatches);
+router.use('/:kbMatch/kb-matched-statements', kbMatchedStatements);
+
```

Comment:
this means there's no way to get a list of kbmatchedStatements for a patient directly, correct?